presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/iceberg.properties

```diff
@@ -0,0 +1,16 @@
+connector.name=iceberg
```

Comment:
We can add similar config files for the other two environments as well (*-with-cross-realm and *-with-wire-encryption).

In addition to this, we would need to also update the docker-compose.yml to mount these configs to the container